private/numerics/functions/elliptic.rkt

```diff
@@ -0,0 +1,241 @@
+#lang racket/base
+
+;; Consult "Numerical Computation of Real or Complex Elliptic
+;; Integrals" Carlson (1994) http://arxiv.org/pdf/math/9409227v1.pdf
+;;
+;; The algorithm in scmutils seems similar to what is in that paper,
+;; but slightly different.
+;;
+;; Consider also exporting under the more familiar names Rf, Rj, Rc,
+;; Rd. Is there any input method that gets those names as subscripts.
+
+(require racket/math)
+(require racket/fixnum)
+
+(define machine-ε
+  (let loop ([ε 1.0])
+    (if (= 1.0 (+ ε 1.0))
+        (* 2 ε)
+        (loop (/ ε 2)))))
+
```

Comment:
That's probably good but I don't really trust parameters. Racket searches the stack frames to find the current binding.

Instead I'd rather have a constant in `main.rkt` and functions just use that as the default behavior in optional arguments (i.e. we don't need the full power of parameters).
```
(define (fn #:epsilon [eps machine-e] . args)
 ...))
```
Seems like the `#:` has to be declared first, but the call-site is flexible.
```
(fn 1 2 3)
(fn 1 2 3 #:eps 'a)
(fn #:eps 'a 1 2 3)
(fn 1 2 #:eps 'a 3)
```